TAC/Meetings/2022/2022-06-16/CCC_TAC_Minutes-2022-06-16.md

```diff
@@ -0,0 +1,145 @@
+* **Recordings**: [YouTube TAC Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
+
+## Links
+
+* **Code of Conduct**: [code-of-conduct.confidentialcomputing.io](https://code-of-conduct.confidentialcomputing.io)
+* **CCC Charter**: [charter.confidentialcomputing.io](https://charter.confidentialcomputing.io)
+* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+Dave Thaler (DT) opened the call at 7:05am PT. He welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation.
+
+DT reviewed the agenda and asked for any additions.
+
+Eric Voit (EV) brought up that Gramine had asked LF Creative Services to redo their website, and they were quoted $5K. They are not pursuing the redo because they feel that this cost is too high. The TAC had approved $1K. 
+
+### Roll Call / Attendance
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* **Accenture** - Giuseppe Giordano *
+* **Ant Group** - Hongliang Tian (Tate)
+* **Arm** - Thomas Fossati * / Michael Lu
+* **Google** - Iulia Ion
+* **Huawei** - Zhipeng (Howard) Huang
+* **Intel** - Dan Middleton * / Simon Johnson
+* **Meta** - Eric Northup * / Shankaran Gnanashanmugam
+* **Microsoft** - Dave Thaler (TAC chair) *
+* **Red Hat** - Lily Sturmann * / Dimitrios Pendarakis
+
+" * " indicates attending
+
+#### Other Attendees
+* Eric Voit (Cisco)
+* Mike Bursell (Profian)
+* Alec Fernandez (Microsoft)
+* Matthieu Legre (Cysec)
+* Mark Novak (JP Morgan Chase)
+* Jethro Beekman (Fortanix)
+* Ravi Sahita (Rivos)
+* Xinxin Fan (Iotex.io)
+* Henk Birkholz (Fraunhofer)
+* Penglin Yang (China Mobile)
+* Naveen Cherukuri (NVIDIA)
+* Vikas Bhatia (Microsoft)
+* Steve Van Lare (Anjuna)
+* Mark Novak (JP Morgan Chase)
+* Simonn Johnson (Intel)
+* Suhas Hegde (Ruby Protocol)
+* Ionut Mihalcea (Arm)
+* Kurt Taylor (Linux Foundation)
+
+#### Not attending 
+* Nagaraju (Intel)
+* Megan Collier (Investment Group)
+* Nick Vidal (Profian)
+* Shalom Shefa (Cisco)
+* Thomas Hardjono (MIT)
+* Hannes Tschofenig (Arm)
+* Helen Lau (Linux Foundation)
```

Comment:
What is the purpose of a "not attending" section?  Anyone on lines 42-50 without a `*` is also not attending....

Reply:
yeah I'd discourage trying to track who isn't in attendance.